### PR TITLE
Clear intepreter stack after pop

### PR DESCRIPTION
**Pull request by slozier**
See https://github.com/IronLanguages/ironpython2/issues/273

### CODE REVIEW

**Comment by slide** [comment]
Is there any sort of test we could add here? I don't have anything specific in mind, it would just be nice to imrove the dlr testing.

**Comment by slozier** [comment]
I have no idea, does the DLR have a builtin language we can use for testing?

**Comment by slide** [comment]
We could add some nunit stuff for testing. We run some tests, but they are just programs that are run.

**Comment by slozier** [comment]
Hmm, interesting the test failed on Travis for netcoreapp. Not sure how I'll go about figuring that one out...

File: Tests/Microsoft.Dynamic.Test/InterpreterTest.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿using NUnit.Framework;
+using System;
+using System.Linq.Expressions;
+
+namespace Microsoft.Dynamic.Test {
+    [TestFixture]
+    public class InterpreterTest {
+        [TestFixture]
+        public class MathUtilsTest {
```
Comment (gpetrou) [inline Tests/Microsoft.Dynamic.Test/InterpreterTest.cs]: Is this needed or a copy/paste error?

Comment (slozier) [reply Tests/Microsoft.Dynamic.Test/InterpreterTest.cs]: Yes, good catch.

**Pull request merged**